run_dir/static/js/fc_trend_plot.js

```diff
@@ -281,26 +297,42 @@ function color_by_chemistry(chem){
     var id = Math.round(window.current_chemistries_list.indexOf(chem));
 	   return current_color_schemes[3](id).hex();
 	}
-//Workaround to get the NovaSeq colors to work  
+//Workaround to get the colors of the chemistry version view to work  
 function color_by_s4(chem){
   var id = Math.round(window.current_chemistries_list.indexOf(chem));
-   return current_color_schemes[4](id).hex();  
+   return current_color_schemes[4](id).hex();
   }
-    
+
 function color_by_s2(chem){
   var id = Math.round(window.current_chemistries_list.indexOf(chem));
    return current_color_schemes[5](id).hex();
   }
 
 function color_by_s1(chem){
   var id = Math.round(window.current_chemistries_list.indexOf(chem));
-   return current_color_schemes[6](id).hex();  
+   return current_color_schemes[6](id).hex();
 }
-        
+
 function color_by_sp(chem){
   var id = Math.round(window.current_chemistries_list.indexOf(chem));
    return current_color_schemes[7](id).hex();  
 }
+function color_by_v2(chem){
```

Comment:
Might we not have just one of there 'color_by' functions as they seem to be doing the same thing? You could have a dictionary in it mapping id to the hardcoded number.

Reply:
hmm I'll have to look into it 👍

Reply:
done!